Applications/Tizen_CAPI/Tizen_CAPI_config.ini

```diff
@@ -0,0 +1,45 @@
+# Network Section : Network
+[Network]
+Type = NeuralNetwork	# Network Type : Regression, KNN, NeuralNetwork
+Layers = inputlayer \
+	 outputlayer	#Layers of Neuralnetwork
+Learning_rate = 0.0001 	# Learning Rate
+Decay_rate = 0.96	# for the decay_rate for the decayed learning rate
+Decay_steps = 1000       # decay step for the exponential decayed learning rate
+Epoch = 10		# Epoch
+Optimizer = adam 	# Optimizer : sgd (stochastic gradien decent),
+ 	    		#             adam (Adamtive Moment Estimation)
+Cost = cross  		# Cost(loss) function : msr (mean square root error)
+                        #                       cross ( for cross entropy )
+Weight_Decay = l2norm
+weight_Decay_Lambda = 0.005
```

Comment:
I want you to keep the consistency  by replacing "**w**eight_Declay_Lambda" with "**W**eight_Declay_Lambda".

Reply:
Yes. Actually, nntrainer is not case sensitive.